workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/camunda/UtilityFunctionsMapper.java

```diff
@@ -67,6 +76,49 @@ public static String escape(String s) {
     return new String(JsonStringEncoder.getInstance().quoteAsString(s));
   }
 
+  public static String encodeRawQuery(String url) throws UnsupportedEncodingException {
+    return encodeRawQuery(url, StandardCharsets.UTF_8.name());
+  }
+
+  public static String encodeRawQuery(String url, String encoding)
+      throws UnsupportedEncodingException {
+    UriComponents uriComponents = UriComponentsBuilder.fromUriString(url).build();
+    MultiValueMap<String, String> queryParamsMap = uriComponents.getQueryParams();
+
+    String rawQuery = uriComponents.getQueryParams()
+        .keySet()
+        .stream()
+        .map(key ->
+            key + "=" + queryParamsMap.get(key).get(0))
+        .collect(Collectors.joining("&"));
+
+    rawQuery = URLEncoder.encode(rawQuery, encoding);
+
+    StringBuilder encodedBuilder = new StringBuilder(uriComponents.getScheme() + "://" + uriComponents.getHost());
```

Comment:
aren't there more components inside an URL? that we could loose here? like basic auth for instance